lib/std/c/freebsd.zig

```diff
@@ -20,6 +20,7 @@ const timespec = std.c.timespec;
 const uid_t = std.c.uid_t;
 const sf_hdtr = std.c.sf_hdtr;
 const clockid_t = std.c.clockid_t;
+const timerfd_clockid_t = std.c.timerfd_clockid_t;
```

Comment:
Hmm, is this actually used? Doesn't seem like it based on the code in `c.zig`.

```suggestion
```

Reply:
no, it's not used